cli/src/semgrep/core_runner.py

```diff
@@ -486,7 +485,6 @@ def __init__(
         self._binary_path = engine_type.get_binary_path()
         self._jobs = jobs or engine_type.default_jobs
         self._engine_type = engine_type
-        self._run_secrets = engine_type
```

Comment:
Should the removed arg be used here? This seems like it may have been a typo but I believe these may be sent as metrics.

Reply:
I was going thru this with @akuhlens and he didn't think it was needed

Reply:
Sounds good; I'd defer to that.